src/Bicep.Core/Emit/TemplateWriter.cs

```diff
@@ -509,6 +520,7 @@ private void EmitModule(JsonTextWriter jsonWriter, ModuleSymbol moduleSymbol, Ex
             emitter.EmitProperty("type", NestedDeploymentResourceType);
             emitter.EmitProperty("apiVersion", NestedDeploymentResourceApiVersion);
 
+            body = EvaluateDecorators(moduleSymbol.DeclaringModule, (ObjectSyntax)body, moduleSymbol.Type);
```

Comment:
It would be nice to rename `EvaluateDecorators` to something which implies it's appending to `body` - since that's not super clear at first when viewing this code.

What about `AddDecoratorsToBody(...)`?